MANIFEST.in

```
@@ -1,3 +1,4 @@
 include README.rst
 include LICENSE.txt
-include *jdcal.py
+include src/jdcal/*.py
```

This should be included automatically by setuptools. We only need to mention the test files:

```suggestion
include tests/*.py
```